src/modules/mount/main.py

```diff
@@ -321,6 +327,10 @@ def run():
         return (_("Configuration Error"),
                 _("No partitions are defined for <pre>{!s}</pre> to use.").format("mount"))
 
+    # swap
+    swap_partitions = [p['device'] for p in partitions if p['fs'] == 'linuxswap' ]
```

Comment:
This is still missing the check for claimed.

Reply:
The presence or absence of CLAIMED should not be a concern.
This is because whether a swap partition is present or not, whether it is configured automatically or manually, is a case-by-case basis.

Reply:
Whether in automatic or manual configuration, if a swap partition exists, it can be enabled to deter the occurrence of OOM killers on low-spec machines.

Reply:
You have to do *something* to ensure that you are not enabling a swap partition that hasn't been identified for use in the installer. 

You can't just enable any swap partition because it may have data that shouldn't be overwritten such as resume data for another distro.

Reply:
Of course, if you were going to do it in earnest, swapfile, and zram would be a consideration, and you should also add a command to exclude the oom killer.
But for now, just enabling the swap partition will reduce the likelihood of the installer being forced to quit in the middle of the install on many low-spec machines.

Reply:
Again, the issue is that you can't enable a swap partition because it exists.  That can cause breakage.

You can only enable a swap partition if it has been selected to be used with the install.

Reply:
```
swap_partitions = [p['device'] for p in partitions if ( p['fs'] == 'linuxswap' and p.get("claimed", None) )]
```

Reply:
And if you are going to check for `claimed`, you need to check that it is `true`, as you suggested here, not that it is `false`, as I have seen in several previous iterations of your patch.